src/components/Details.jsx

```diff
@@ -39,9 +38,9 @@ const Details = () => {
   const [balance, setBalance] = useState("Please connect wallet");
   const [deposit, setDeposit] = useState("Please connect wallet");
   const [userInfo, setUserInfo] = useState({
-    "first_name":"none",
-    "last_name":"none",
-    "address":"none"
+    first_name: "none",
+    last_name: "none",
+    address: "none",
```

Comment:
since you add 3 more fields in getUserInfo(), you might want to add here too

Reply:
Fixed, thank you!